skimage/measure/_regionprops.py

```diff
@@ -397,8 +405,8 @@ def image_convex(self):
     @property
     def coords(self):
         indices = np.nonzero(self.image)
-        return np.vstack([indices[i] + self.slice[i].start
-                          for i in range(self._ndim)]).T
+        return np.vstack([(indices[i] + self.slice[i].start) * s
+                          for i, s in zip(range(self._ndim), self._spacing)]).T
```

Comment:
imho these coordinates are for indexing into the image and should not be scaled.

Perhaps we should have `coords` and `coords_scaled`, or `indices` and `coords`? And the docstring should state which property is scaled and which is not...

Reply:
Introduced `coords_scaled`.